src/components/navigation.tsx

```diff
@@ -28,73 +43,182 @@ export default function Navigation({locale, navigationLabels}: Props) {
+                                            : 'font-normal',
+                                          'ml-3 block truncate'
+                                        )}
+                                      >
+                                        {language.name}
+                                      </span>
+                                    </div>
+
+                                    {selected ? (
+                                      <span
+                                        className={classNames(
+                                          active
+                                            ? 'text-white'
+                                            : 'text-indigo-600',
+                                          'absolute inset-y-0 right-0 flex items-center pr-4'
+                                        )}
+                                      >
+                                        <CheckIcon
+                                          className="h-5 w-5"
+                                          aria-hidden="true"
+                                        />
+                                      </span>
+                                    ) : null}
+                                  </>
+                                )}
+                              </Listbox.Option>
+                            </Link>
+                          ))}
+                        </Listbox.Options>
+                      </Transition>
+                    </div>
+                  </>
+                )}
+              </Listbox>
             </div>
           </div>
-
-          <Disclosure.Panel className="sm:hidden">
-            <div className="space-y-1 pt-2 pb-3">
-              {navigation.map(item => {
-                const current = item.href === pathname;
-                return (
-                  <Disclosure.Button
-                    key={item.name}
-                    as={Link}
-                    href={`/${locale}${item.href}`}
-                    className={classNames(
-                      current
-                        ? 'bg-indigo-50 border-indigo-500 text-indigo-700'
-                        : 'border-transparent text-gray-600 hover:bg-gray-50 hover:border-gray-300 hover:text-gray-800',
-                      'block pl-3 pr-4 py-2 border-l-4 text-base font-medium'
+        </nav>
+      </header>
+      <Disclosure as="nav" className="border-b border-gray-200 bg-white">
+        {({open}) => (
+          <>
+            <div className="mx-auto max-w-7xl px-4 sm:px-6 lg:px-8">
+              <div className="flex h-16 justify-between">
+                <div className="inline-flex items-center px-1 pt-1">
+                  {/* TODO: Dataset Browser logo */}
+                </div>
+                <div className="flex">
+                  <div className="hidden sm:-my-px sm:flex sm:space-x-8">
+                    {navigation.map(item => {
+                      const current = item.href === pathname;
```

Comment:
Minor: what is `current`? Is it possible to add a name to it (e.g. 'currentPathname`)?

Reply:
Changed to `isCurrentPathname` so it's clear it's a boolean.